File: RPG_Game/Assets/Scripts/Config/CameraSettings.js
#pragma strict

public class CameraSettings extends MonoBehaviour
{
	private static var instance : CameraSettings;
	
	public static function Instance() : CameraSettings
	{
		return instance;
	}
	
	// General Settings
	var xCamSpeed : float;
	var yCamSpeed : float;
	var camDistance : float;
	
	// Zone Settings
	var relativeCamPos : Vector3;
	var relativeLookPos : Vector3;
	
	function Awake()
	{
		instance = this;
		
		// Camera
		xCamSpeed = 2.5f;
		yCamSpeed = 2.5f;
		camDistance = 6.0f;
	}
}

File: RPG_Game/Assets/Scripts/Config/CameraSettings.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class CameraSettings : MonoBehaviour {

    private static CameraSettings instance;
	
	public static CameraSettings Instance()
	{
		return instance;
	}

    // General Settings
    public float xCamSpeed;
	public float yCamSpeed;
	public float camDistance;

    // Zone Settings
    public Vector3 relativeCamPos;
    public Vector3 relativeLookPos;
	
	void Awake()
    {
        instance = this;

        // Camera
        xCamSpeed = 2.5f;
        yCamSpeed = 2.5f;
        camDistance = 6.0f;
    }
}
